### DISCUSSION TITLE
外文资料的书面翻译中摘要的关键词无法正确显示

### DISCUSSION THREAD

**Discussion by DKFMaster** [tuna/thuthesis#800] [Q&A] [answered]
在编辑本科毕设“外文资料的书面翻译”中，摘要中的关键词无法显示。请问如何处理？代码如下，其中\thusetup{}内的内容无法显示。
\begin{abstract}
	123
	\thusetup{
		keywords = {1, 2, 3},
	}
\end{abstract}

**Comment by zepinglee** [comment, marked as answer]
这里的关键词直接手写。毕竟“外文资料的书面翻译”没有规定关键词的格式，所以没必要用 `\thusetup` 了。
